src/__tests__/Calculate.test.js

```diff
@@ -0,0 +1,33 @@
+import calculate from '../logic/Calculate';
+
```

Comment:
- [ ] As mentioned by the previous reviewer, let us kindly test for the button `'.'`